### PR TITLE
Mosaic utility

### PR DESCRIPTION
**Pull request by matteo-gildone**
moved utility classes from Mosaic

### CODE REVIEW

File: packages/global-context/README.md [hunk 1]
```diff
@@ -2,11 +2,8 @@
 
 Set the context (baseline) for branded products. This package is used as a template and your product should use the context for a specific brand that inherits from this.
 
-- [Breakpoints](#breakpoints)
-- [Colors](#colors)
-	- [Greyscale](#greyscale)
-	- [Color](#color)
-	- [Foreground color](#foreground-color)
+-   [Breakpoints](#breakpoints)
+-   [Colors](#colors) - [Greyscale](#greyscale) - [Color](#color) - [Foreground color](#foreground-color)
```
Comment (alexkilgour) [inline packages/global-context/README.md]: Can you link to the layers section from the nav here

File: packages/global-context/README.md [hunk 2]
```diff
@@ -2,11 +2,8 @@
 
 Set the context (baseline) for branded products. This package is used as a template and your product should use the context for a specific brand that inherits from this.
```
Comment (alexkilgour) [inline packages/global-context/README.md]: something seems to have messed with all the formatting here, possibly your editor. When I view the file a bunch of the layout is broken from all the changes to this file - for example, the navigation below is broken

File: packages/global-context/scss/20-functions/layers.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@function z($layer-name, $layer-variant: null, $map: $z-layers) {
+	@if not map-has-key($map, $layer) {
+		@warn "No z-index found in $z-layers map for `#{$layer}`. Property omitted.";
+	}
+
+	@if ($layer-variant != null) {
+		// map inception
+		@return map-get(map-get($map, $layer-name), $layer-variant);
+	} @else {
```
Comment (alexkilgour) [inline packages/global-context/scss/20-functions/layers.scss]: https://travis-ci.org/springernature/frontend-global-toolkit/builds/548625298#L496

File: packages/global-context/README.md (hunk 2 above)
Comment (matteo-gildone) [reply packages/global-context/README.md]: I'm working on fixing it right now

Comment (matteo-gildone) [reply packages/global-context/README.md]: fixed

File: packages/global-context/README.md (hunk 1 above)
Comment (matteo-gildone) [reply packages/global-context/README.md]: added

File: packages/global-context/scss/20-functions/layers.scss (hunk 3 above)
Comment (matteo-gildone) [reply packages/global-context/scss/20-functions/layers.scss]: fixed

**Pull request merged**